src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/af1ae333-418c-4349-ade0-665e020ee4d1)

Reply:
fixed